components/automate-chef-io/content/docs/eas_applications.md

```diff
@@ -0,0 +1,160 @@
+
+To configure the threshold for removing a disconnected service, you can do this via the UI or the API directly. The threshold is the time since it was marked as disconnected. You can also disable removal by setting running to false. To access the configuration settings in the UI, go to the settings tab, then Data Lifecycle on the sidebar. Under service groups you will find settings for Marking and Removing disconnected services.
+
+Below is an example of using the API directly:
+
+```bash
+curl -sSX POST "https://automate-url/apis/v0/retention/service_groups/delete_disconnected_services/config" -d
+'{
+  "threshold": "1d",
+  "running":true
+}'
+-H "api-token: $TOKEN"
+```
+
+## Configure encrypted communication
+
+### Disable TLS For the Event Stream Protocol
+If you do not wish to configure encrypted communication between habitat and automate you can disable TLS.
+
+On the host where Chef Automate is deployed, create a file with the following content:
+```
+[event_gateway]
+  [event_gateway.v1]
+    [event_gateway.v1.sys]
+      [event_gateway.v1.sys.service]
+        disable_frontend_tls = true
+```
+
+Save the file with any name, then run chef-automate config patch FILENAME to apply the configuration change. You should see output similar to the following:
+
+Updating deployment configuration
+
+Applying deployment configuration
+  Started event-gateway
+
+### Configure automate for encrypted communication
+
+By default frontend TLS is enabled. If it was previously disabled it can be re-enabled. On the host where Chef Automate is deployed, create a file with the following content:
+```
+[event_gateway]
+  [event_gateway.v1]
+    [event_gateway.v1.sys]
+      [event_gateway.v1.sys.service]
+        disable_frontend_tls = false
+```
+
+Save the file with any name, then run chef-automate config patch FILENAME to apply the configuration change. You should see output similar to the following:
+
+Updating deployment configuration
+
+Applying deployment configuration
+  Started event-gateway
+
+Once you have done that, retrieve the automate frontend TLS key to use for configuring your habitat supervisor.
+
+You can cat the cert file that is located at `/hab/svc/automate-load-balancer/data/<servername>.crt` to retrieve the current frontend tls certificates.
+
+You can also change the automate frontend key by following the configuration instructions referenced
+(https://automate.chef.io/docs/configuration/#load-balancer-certificate-and-private-key "here").
+
+### Configure habitat supervisor for encrypted communication
+
+There are three ways to share the automate certificate with the habitat supervisor to use when connecting to automate.
+
+1) Pass the certificate file as a parameter to the --event-stream-server-certificate flag.
+
+```
+HAB_FEAT_EVENT_STREAM=1 hab sup run \
```

Comment:
Do we know when the hab feature flag will no longer be needed?

Reply:
@davidMcneil is going to bring it up with the hab team.